daemons/compute/server/servers/server_default.go

```diff
@@ -312,7 +312,10 @@ func (s *defaultServer) Create(ctx context.Context, req *pb.DataMovementCreateRe
 	// Authentication
 	userId, groupId, err := auth.GetAuthInfo(ctx)
 	if err != nil {
-		return nil, err
+		return &pb.DataMovementCreateResponse{
+			Status:  pb.DataMovementCreateResponse_FAILED,
+			Message: "Error authenticating: " + err.Error(),
```

Comment:
As long as you are here, modify all err.Error() responses with unique message prefixes.